app/managers.py

```diff
@@ -0,0 +1,49 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            "(first_name, last_name) VALUES (?,?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        actors_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [Actor(*row) for row in actors_cursor]
+
+    def update(
+            self,
+            id_to_update: int,
+            new_first_name: str,
+            new_last_name: str
+    ) -> None:
+        self._connection.execute(
+            f"UPDATE {self.table_name} "
+            "SET first_name = ?, last_name = ? "
+            "WHERE id = ?",
+            (new_first_name, new_last_name, id_to_update)
+        )
+        self._connection.commit()
+
+    def delete(self, id_to_delete: int) -> None:
+        self._connection.execute(
+            f"DELETE FROM {self.table_name} WHERE id = ?",
+            (id_to_delete,)
+        )
+        self._connection.commit()
+
+
+if __name__ == "__main__":
+    manager = ActorManager()
+    manager.delete(id_to_delete=1)
+    print(manager.all())

```

Comment:
You have to test your `ActorManager` in `main.py`
https://github.com/prykhodkom/py-actor-manager/tree/develop#:~:text=Test%20different%20methods%20in%20the%20main.py%20module.